npm/private/npm_translate_lock.bzl

```diff
@@ -499,6 +501,12 @@ def npm_translate_lock(
 
         pnpm_version: pnpm version to use when generating the @pnpm repository. Set to None to not create this repository.
 
+            Use `use_pnpm` for bzlmod.
+
+        use_pnpm: label of the pnpm extension to use
+
+            Use `pnpm_version` for non-bzlmod.
```

Comment:
If this is unset, the version of pnpm comes from the rules_js MODULE.bazel file? Should mention what happens when it is left unset.

Reply:
Should mention that the label should be fully qualified as `"@repository_name//:package/bin/pnpm.cjs"` or else take a repository name and add the `//:package/bin/pnpm.cjs` implicitly. Maybe support both so if the label is _not_ fully qualified, for example just `@pnpm`, then `//:package/bin/pnpm.cjs` is added or else it is not.

Reply:
Testing for the existence of `//` in the string ought to be enough to assert if you need to append `//:package/bin/pnpm.cjs` or not.

Reply:
Since bzlmod extensions are not macros I don't think this is possible. By the time we have access to the `use_pnpm` it has already been evaluated so the default target will be there.

Reply:
If we find a way to do it I think it will be (almost 😅) non-breaking so I think we can keep it as-is for now where the user must specify the full path.